pkg/dsync/drwmutex.go

```diff
@@ -237,8 +236,8 @@ func lock(ds *Dsync, locks *[]string, id, source string, isReadLock bool, lockNa
 		done := false
 		timeout := time.After(DRWMutexAcquireTimeout)
 
-		dquorum := int(len(restClnts)/2) + 1
-		dquorumReads := int(math.Ceil(float64(len(restClnts)) / 2.0))
+		dquorumReads := (len(restClnts) + 1) / 2
+		dquorum := dquorumReads + 1
```

Comment:
Same reasoning we use to write atleast one extra parity, assumption is always hold +1 extra write lock to keep the symmetry. Otherwise it's hard to reason when even nodes occur we hold +1 but with odd node we relax. To keep the overall assumptions same.

Reply:
Yeah, let's keep the extra.